.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Build Release Artifacts
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: 14
+    - name: Checkout Data-Prepper
+      uses: actions/checkout@v2
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
```

Comment:
Is this step needed? [gradlew](https://github.com/opensearch-project/data-prepper/blob/main/gradlew) already has execute permissions.

Reply:
I asked this same question on another PR recently. It appears all of our GitHub Actions do this. I'll try to experiment to see if this really matters or not.

Reply:
I created a new PR that removes this step for all builds:

https://github.com/opensearch-project/data-prepper/pull/1028